### PR TITLE
(example) DayOfWeekPicker: new widget and example

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
[Issue] https://github.com/Samsung/TAU/issues/1075
[Problem] Day of week picker in sample app
[Solution]
  - New widget DayOfWeek has been implemented
    in ScheduleApp application
  - Option SelectAll in AppBar has been required fix

[Demo]
obraz

Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

### CODE REVIEW

File: examples/mobile/ScheduleApp/js/DayOfWeekPicker.js [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+		var self = this;
+
+		self.element.removeEventListener("change", self, true);
+	};
+
+	/**
+	 * Destroy DayOfWeekPicker component
+	 * @method _destroy
+	 * @param {HTMLElement} element
+	 * @return {HTMLElement} Returns built element
```
Comment (HunseopJeong) [inline examples/mobile/ScheduleApp/js/DayOfWeekPicker.js]: There is no return value.

File: examples/mobile/ScheduleApp/js/DayOfWeekPicker.js [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+	*/
+	prototype._onChange = function () {
+		var self = this;
+
+		events.trigger(self.element, EVENT_TYPE.CHANGE, {value: self._getValue()});
+	};
+
+	/**
+	 * Widget common event handler
+	 * @method handleEvent
```
Comment (HunseopJeong) [inline examples/mobile/ScheduleApp/js/DayOfWeekPicker.js:146]: Need a param for event

File: examples/mobile/ScheduleApp/css/DayOfWeekPicker.css [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    content: "T";
+}
+.ui-day-of-week-picker input:nth-child(4)::after {
+    content: "W";
+}
+.ui-day-of-week-picker input:nth-child(5)::after {
+    content: "T";
+}
+.ui-day-of-week-picker input:nth-child(6)::after {
+    content: "P";
```
Comment (HunseopJeong) [inline examples/mobile/ScheduleApp/css/DayOfWeekPicker.css]: "F" is correct.

File: examples/mobile/ScheduleApp/js/DayOfWeekPicker.js (hunk 1 above)
Comment (TomaszLukawskiSam) [reply examples/mobile/ScheduleApp/js/DayOfWeekPicker.js]: Done

File: examples/mobile/ScheduleApp/js/DayOfWeekPicker.js (hunk 2 above)
Comment (TomaszLukawskiSam) [reply examples/mobile/ScheduleApp/js/DayOfWeekPicker.js:146]: Done

File: examples/mobile/ScheduleApp/css/DayOfWeekPicker.css (hunk 3 above)
Comment (TomaszLukawskiSam) [reply examples/mobile/ScheduleApp/css/DayOfWeekPicker.css]: Done.
Thank you for review :)

File: examples/mobile/ScheduleApp/js/DayOfWeekPicker.js [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+				checkbox.checked = value.indexOf(key) > -1;
+			});
+		} else {
+			ns.warn("Widget accepts string value like '0,1,5'.");
+		}
+	};
+
+	/**
+	* Get value of widget
+	* @method _getValue
```
Comment (HunseopJeong) [inline examples/mobile/ScheduleApp/js/DayOfWeekPicker.js:116]: Need a result value.

Comment (TomaszLukawskiSam) [reply examples/mobile/ScheduleApp/js/DayOfWeekPicker.js:116]: Done :)

**Review by HunseopJeong — APPROVED** [review]
LGTM!

**Pull request merged**